apps/console/src/features/roles/components/edit-role/edit-role-common/permissions-list.tsx

```diff
@@ -45,36 +45,37 @@ interface PermissionsListPropsInterface extends  IdentifiableComponentInterface
      * Callback to handle API resource removal.
      */
     onChangeScopes: (apiResource: APIResourceInterface, scopes: ScopeInterface[]) => void;
+    /**
+     * Whether the Autocomplete field has an error or not.
```

Comment:
Let's not tie the implementation to comments. How about something like "Whether the permissions list component has an error"?

PS: Let's update the other comments as well.

Reply:
Addressed: 16ae921f4144efa91f6b4f93e7bb21922243052b